apps/SageAccountWeb/src/style/_variables.scss

```diff
@@ -1,4 +1,4 @@
 // Variables used in Portals SCSS should be defined in this file.
 
-$primary-action-color: #2a2a2a !default;
+$primary-action-color: #395979 !default;
```

Comment:
This styles the links in the EntityFinder. Use the Synapse primary color instead of a dark gray color.